lib/checks/canFindRoots.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+const resolvesToConstant = require('./resolvesToConstant.js');
+
+function canFindRoots(equation) {
+
+  // Equation should be in the form of factor * factor = 0, factor^power = 0, or identifier = 0
+  // e.g (x - 2)^2 = 0, x(x + 2)(x - 2) = 0
+  const left = equation.leftNode;
+  const right = equation.rightNode;
+
+  const zeroRightSide = Node.Type.isConstant(right)
```

Comment:
Ahhh yes! I tried adding that helper, but ran into some issues. I added an extra param for validating a constant, but it seems like somewhere in mathsteps we are already calling `Node.Type.isConstant` with an extra param. Haven't fully investigated, but Evy said it would be fine if we push that fix into a separate PR.